drivers/hmis/app/models/hmis/file.rb

```diff
@@ -0,0 +1,31 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+class Hmis::File < GrdaWarehouse::File
+  include ClientFileBase
+
+  SORT_OPTIONS = [
+    :date_created,
+    :date_updated,
+  ].freeze
+
+  self.table_name = :files
+  belongs_to :enrollment, class_name: '::Hmis::Hud::Enrollment', optional: true
+  belongs_to :client, class_name: '::Hmis::Hud::Client'
+
+  def self.sort_by_option(option)
+    raise NotImplementedError unless SORT_OPTIONS.include?(option)
+
+    case option
+    when :date_created
+      order(arel_table[:created_at].asc.nulls_last)
+    when :date_updated
+      order(arel_table[:updated_at].asc.nulls_last)
```

Comment:
Might also want to be able to sort by effective date or expiration date too.

Reply:
I thought we can skip them for now since they probably won't have their own table columns because they are file-tag-dependent

Reply:
Skipping these for now then